### PR TITLE
chore: fix some more initialisms

### PR DESCRIPTION
**Pull request by blgm**
Follows on from commits bdd1de9, f568553, and 459a72f

https://github.com/golang/go/wiki/CodeReviewComments#initialisms
Go guidelines (and statickcheck rule ST1003) recommend using ID, JSON,
etc... rather than Id, Json, etc...

### Checklist:

* ~~[ ] Have you added or updated tests to validate the changed functionality?~~
* ~~[ ] Have you added Draft Release Notes in `docs/draft-release-notes.md`?~~
* [X] Have you followed the [Conventional Commits specification](https://www.conventionalcommits.org/en/v1.0.0/#summary)?

### CODE REVIEW

**Comment by zucchinidev** [comment]
Great job @blgm, thank you very much!

File: db_service/models/historical_db.go [hunk 1]
```diff
@@ -135,24 +135,24 @@ type ServiceInstanceDetailsV3 struct {
...
 	Location     string
-	Url          string
+	URL          string
 	OtherDetails []byte `gorm:"type:blob"`
 
-	ServiceId        string
-	PlanId           string
-	SpaceGuid        string
-	OrganizationGuid string
+	ServiceID        string
```
Comment (pivotal-marcela-campo) [inline db_service/models/historical_db.go:141]: Just wondering how does GORM treats this change. I am asuming field names will continue to be the same and `ServiceID` will be changed to `service_id` in the database as it was before. Is that the case?

**Review by pivotal-marcela-campo — COMMENTED** [review]
Looks ok, just have a question about the change in field names in models.

File: db_service/models/historical_db.go (hunk 1 above)
Comment (blgm) [reply db_service/models/historical_db.go:141]: That's a good question. I looked at the GORM tests and think it should be ok: https://github.com/go-gorm/gorm/blob/373bcf7aca01ef76c8ba5c3bc1ff191b020afc7b/schema/naming_test.go#L10-L26

**Review by pivotal-marcela-campo — APPROVED** [review]

**Pull request merged**